dotnet/dotnet-guestbook/kubernetes-manifests/guestbook-backend.deployment.yaml

```diff
@@ -30,4 +34,4 @@ spec:
         - name: PORT
           value: "8080"
         - name: GUESTBOOK_DB_ADDR
-          value: dotnet-guestbook-mongodb:27017
\ No newline at end of file
+          value: dotnet-guestbook-mongodb:27017

```

Comment:
what is the change here?

Reply:
It looks like vim added a newline to the last line.